### DISCUSSION TITLE
Voice Messages

### DISCUSSION THREAD

**Discussion by juanpdebonis** [cstayyab/WALC#199] [Troubleshooting] [answered]
The app says that doesn't detect a microphone on the computer and therefore doesn't allow me to send voice messages. Nonetheless, great job people! keep it up!

**Comment by kawak1320** [comment]
hey! maybe you should report this as an issue? or check your mic in your system? i can send voice messages!

**Reply by juanpdebonis** [reply]
I didn't knew. After a reboot keeps happening, so I think it's a bug.

**Comment by juanpdebonis** [comment, marked as answer]
I resolved by looking on the issues. It was something linked to snap not giving the permission to my mic. Solved.
